web/src/games/tictactoe/index.ts

```diff
@@ -6,6 +6,7 @@ import instructions from './instructions.md';
 export const tictactoeGameDef: IGameDef = {
   code: 'tictactoe',
   name: 'Tic-Tac-Toe',
+  contributors: ['flamecoals'],
```

Comment:
? 

[Add Tic Tac Toe Game by jasonharrison · Pull Request #69 · freeboardgames/FreeBoardGames.org](https://github.com/freeboardgames/FreeBoardGames.org/pull/69)

Reply:
Fixed, sorry, github history was missing this for old code and I just  guessed, I was going to double check before sending to review.